src/KafkaFlow.Retry.SqlServer/KafkaFlow.Retry.SqlServer.csproj

```diff
@@ -36,7 +36,7 @@
   </ItemGroup>
 
   <ItemGroup>
-    <PackageReference Include="System.Data.SqlClient" Version="4.8.2" />
+    <PackageReference Include="System.Data.SqlClient" Version="4.8.5" />
```

Comment:
Can we change to `Microsoft.Data.SqlClient` instead?

https://www.nuget.org/packages/Microsoft.Data.SqlClient/5.1.2

The reason is that the `System.Data.SqlClient` development has moved to this new one, and the older one is kept only in maintenance mode.

I think the change is mostly on namespaces (pretty straightforward) but here is the [migration guide](https://github.com/dotnet/SqlClient/blob/main/porting-cheat-sheet.md).

Reply:
On it.

Reply:
✅